contracts/Oracle.sol

```diff
@@ -146,10 +146,16 @@ contract Oracle is Ownable {
      * @return True if locking period is over, False if not
      */
     function isLockingPeriodOver(address _asset, uint256 _expiryTimestamp) public view returns (bool) {
-        address pricer = assetPricer[_asset];
-        uint256 lockingPeriod = pricerLockingPeriod[pricer];
+        uint256 price = stablePrice[_asset];
+
+        if (price == 0) {
```

Comment:
I think it's cleaner to use a (uint256 => bool) mapping `isStableAsset` to check the type of asset, instead of price == 0.

Or add a function `isStableAsset()` for this repeating check.

Reply:
I thought about it, IMO feel it is just more code, can still use `0` as non-existent as we will not have a stable price equal to 0

Reply:
How about a function called `isStableAsset()` and in that function check if `stablePrice != 0`?